cli/util/git.go

```diff
@@ -5,12 +5,16 @@ import (
 	"fmt"
 	"os"
 	"os/exec"
+	"regexp"
 	"strings"
 	"unicode"
 
 	"github.com/hashicorp/go-version"
 )
 
+// MinCommitHashLength is the minimum hash length required in git.
+const MinCommitHashLength = 7
```

Comment:
Looks like it is not used out of this package. Let's make it internal.

Reply:
fixed again